src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,6 +111,25 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
+@include breakpoint($small-screen) {
+    // JUST FOR TESTING ******** REMOVE BEFORE PUSHING ----
+    .bdl-CollapsibleSidebar-wrapper.hidden {
+        display: block !important;
+    }
+
+    .is-responsive-web {
+        .bdl-CollapsibleSidebar.is-expanded {
+            width: 100%;
+
+            .bdl-CollapsibleSidebar-nav ul {
+                // On expand, set ul to fullscreen width with no transition for performance.
+                // Use vw because relative parent (custom-scrollbar) is not full screen width
+                width: calc(100vw - (#{$collapsible-side-bar-h-padding} * 2));
```

Comment:
Would it work to disable padding of the parents, and then use `width:100%`? This `width: calc(100vw` is clever, but seems fragile versus the relative `width:100%` approach.

Reply:
The comment above the code explains why I went with this approach. I don't want it to be "clever" and figured that's how it would seem. 

To explain a little further, the relative parent of the ul element is the scrollbar element which has its own dimensions that can't be relied on for getting screen width. Therefore, without using javascript, the only way without modifying the scrollbar (which is probably not possible) is to use vw.

Reply:
I entered these changes manually in the browser:

.bdl-CollapsibleSidebar.is-expanded {
    width: 100%;
}

@media (max-width: 767px)
.bdl-CollapsibleSidebar-wrapper {
    width: 100%;
}

@media (max-width: 767px)
.bdl-CollapsibleSidebar.is-expanded .bdl-CollapsibleSidebar-nav ul {
    width: 100%;
}

And I got the following result:
![Screen Shot 2022-05-04 at 5 04 01 PM](https://user-images.githubusercontent.com/5461045/166825635-f33100c3-cbc6-412c-8588-6d6a54f76ef5.png)
![Screen Shot 2022-05-04 at 5 04 09 PM](https://user-images.githubusercontent.com/5461045/166825624-e069d889-0947-48a9-8f0d-8dba87a747f8.png)

Does that seem right?

Reply:
Oh it's actually not right, because you want small screen-width rather than medium. But anyway, if you just change the breakpoint that I put there from medium to small, will that do it for ya?

Reply:
Yeah, so there's a gotcha with this approach; The animation choppiness due to the ul contents. Setting the ul width to 100% will cause the ul and all of it's content (which is a lot) to "follow" the animation / transition which overloads the GPU I assume and causes choppiness.

Reply:
It does because `.bdl-CollapsibleSidebar-nav ul` has a lot of elements which needs to be resized / recalculated during the layout step in the browser render which is costly. This could possibly be fixed by redesigning the li elements into flex items and reducing the number of els in each li while still keeping accessibility in tact. But that's out of the scope of this ticket.

Reply:
Thanks for your great demo of this today. I now understand the problem, and I like your solution.